### DISCUSSION TITLE
Prism Navigation prevents pop up wait indicator from being displayed

### DISCUSSION THREAD

**Discussion by stevensrf11** [PrismLibrary/Prism#2410] [WPF]
I am doing navigation via a event handler method that is registered for a compositecommand.
When that method is called first a binding is set that is suppose to display a pop up wait indicator.
Next navigation is down.
If I comment out the navigation code the wait indicator is seen.
If I do not comment it out the wait indicator is not seen.

Any one ever have an issue like this?

**Comment by brianlagunas** [comment]
This won't work because nothing is being done async. There is only one UI thread, and showing your IsBusy and navigating both happen on the same UI thread.

**Comment by stevensrf11** [comment]
Awww ok Thanks you for information.

If my sub penny stock ever gets to a penny I will double my monthly funding

 
 

Sent: Wednesday, April 21, 2021 at 9:41 AM

**Reply by brianlagunas** [reply]
![](https://thumbs.gfycat.com/LightheartedFlippantGilamonster-size_restricted.gif)

**Reply by stevensrf11** [reply]
LOL THATS COOL
